sklearn/svm/base.py

```diff
@@ -538,7 +538,7 @@ def decision_function(self, X):
         dec = self._decision_function(X)
         if self.decision_function_shape is None and len(self.classes_) > 2:
             warnings.warn("The decision_function_shape default value will "
-                          "change from 'ovo' to 'ovr' in 0.18. This will change "
+                          "change from 'ovo' to 'ovr' in 0.19. This will change "
```

Comment:
Or wait I think this change was actually done in 0.17 and should be removed in 0.19 only. Il confirm in sometime.